src/client/containers/authentication/authentication.js

```diff
@@ -83,7 +79,11 @@ class Authentication extends Component {
 
 Authentication.propTypes = {
   history: PropTypes.instanceOf(Object).isRequired,
-  postUserData: PropTypes.func.isRequired,
+  onAuth: PropTypes.func,
 };
 
-export default connect(null, { postUserData })(withRouter(Authentication));
+const mapDispatchtoProps = dispatch => ({
+  onAuth: (path, newUser, history) => dispatch(postUserData(path, newUser, history)),
+});
+
+export default connect(null, mapDispatchtoProps)(withRouter(Authentication));
```

Comment:
And of course here with naming of map function

Reply:
done